### PR TITLE
Update contract to provwasm 2.0

### PR DESCRIPTION
**Pull request by jeradhoy**
Updates the contract to provwasm 2.0 in order to get contract routes working with provenance 1.19.

### CODE REVIEW

**Comment by jeradhoy** [comment]
https://app.shortcut.com/figure/story/351144/release-securitization-v2-smart-contract-updates-for-provwasm-2-0

File: crates/contract-controller/Cargo.toml [hunk 1]
```diff
@@ -28,7 +28,7 @@ overflow-checks = true
 
 [features]
 # for more explicit tests, cargo test --features=backtraces
-backtraces = ["cosmwasm-std/backtraces"]
+#backtraces = ["cosmwasm-std/backtraces"]
```
Comment (jtolotti-figure) [inline crates/contract-controller/Cargo.toml:31]: Should this be commented out? If so should we delete it?

Comment (jeradhoy) [reply crates/contract-controller/Cargo.toml:31]: Ahhh, yeah this was deprecated in the update cosmwasm-std version, I'll remove

**Review by chrisbredernitz — APPROVED** [review]

File: crates/contract/src/execute/settlement/withdraw_commitment.rs [hunk 2]
```diff
@@ -1,6 +1,10 @@
 use cosmwasm_std::{Addr, Env, Event, Response, Storage};
-use provwasm_std::{mint_marker_supply, transfer_marker_coins, withdraw_coins};
+// use provwasm_std::{mint_marker_supply, transfer_marker_coins, withdraw_coins};
```
Comment (jtolotti-figure) [inline crates/contract/src/execute/settlement/withdraw_commitment.rs:2]: Probably out to delete this guy as well

Comment (jeradhoy) [reply crates/contract/src/execute/settlement/withdraw_commitment.rs]: Removed!

**Review by jtolotti-figure — APPROVED** [review]

**Pull request merged**